packages/dapi-grpc/protos/platform/v0/platform.proto

```diff
@@ -158,6 +159,37 @@ message GetIdentitiesKeysResponse {
   ResponseMetadata metadata = 3;
 }
 
+message GetProofsRequest {
+  message DocumentProofRequest {
```

Comment:
Shouldn't it be DocumentRequest to conform with IdentityRequest and ContractRequest?

Reply:
fixed